deployment/proxies.rst

```diff
@@ -159,6 +159,20 @@ enough, as it will only trust the node sitting directly above your application
 ranges of any additional proxy (e.g. `CloudFront IP ranges`_) to the array of
 trusted proxies.
 
+Reverse proxy in a subpath / subfolder
+--------------------------------------
+
+If you configure the reverse proxy in such a way that the Symfony application is
```

Comment:
if i understand correctly, this is about the proxy routing to symfony for `/subpath`, but symfony itself running on `/`? in a way this is the only thing that makes sense for this documentation, nonetheless i would make that more explicit. you could start with the example and explicitly explain that the symfony application runs at `/`, and give the example of what absolute paths/urls should be.

Reply:
@jbtronics if possible, please add a phrase or two as a short introduction to the exact problem that this solves, as asked by David. Thanks!